modules/dns_common/dns_common.rb

```diff
@@ -170,10 +170,10 @@ def aaaa_record_conflicts(fqdn, ip)
       end
     end
 
-    def cname_record_conflicts(fqdn, target)
-      current = resolver.getresources(fqdn, Resolv::DNS::Resource::IN::CNAME)
+    def cname_record_conflicts(fqdn, host_alias)
+      current = resolver.getresources(host_alias, Resolv::DNS::Resource::IN::CNAME)
```

Comment:
I don't think this is correct. If I interpret it correctly then ```fqdn``` is the name and ```target``` the value. https://github.com/theforeman/smart-proxy/blob/47732b9ef1375e7bc88b7056f497080477ac2e31/modules/dns/dns_api.rb#L30 passes ```fqdn```, ```value```. Then https://github.com/theforeman/smart-proxy/blob/47732b9ef1375e7bc88b7056f497080477ac2e31/modules/dns_common/dns_common.rb#L45-L46 does the same.